src/Integration.Vsix/SonarAnalyzerManager.cs

```diff
@@ -50,11 +51,19 @@ public class SonarAnalyzerDeactivationManager
             this.workspace = workspace;
             this.activeSolutionBoundTracker = serviceProvider.GetMefService<IActiveSolutionBoundTracker>();
 
+            if (this.activeSolutionBoundTracker == null)
+            {
+                throw new NullReferenceException("Could not get " + nameof(IActiveSolutionBoundTracker));
```

Comment:
Asserting should be enough since we own the service. Otherwise the exception message needs to be localizable (i.e. in resx)